index.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles/styles.css">
+    <title>Fabricio La Torre</title>
+</head>
+<body>
+    <header class="main-header">
+        <div class="container">
+            <h1 class="mh-logo">
+              <a href="#"><img alt="kiwilogo" src="images/kiwilogo.svg"></a>
+                
+            </h1>
+            <nav class="main-nav">
+               
+                <ul class="main-nav-list">
+                  
+                    <li>
+                        <a href="#">Portfolio</a>
+                    </li>
+                    <li>
+                        <a href="#">About</a>
+                    </li>
+                    <li>
+                        <a href="#">Contact</a>
+                    </li>
+                    <li>
+                        <a href="#"><img alt="maillogo" src="images/mail.svg"></a>
+                    </li>
+                </ul>
```

Comment:
- [ ] Please remove the `ul` tag with everything in it and replace it with a hamburger button as seen in the mobile version of the Figma template. Right now we are working on the mobile version and this is only needed in the desktop version of the portfolio 👍